extra-radio/svxlink/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=svxlink
+PKGSEC=hamradio
+PKGDEP="alsa-utils alsa-lib libsigc++ gsm libgcrypt popt tcl speex opus qt-5"
+PKGDES="Echolink Software to provide Ham Radio Voip Operations"
+
+ABTYPE="cmake"
+CMAKE_AFTER=" -DUSE_QT=YES ../src"
```

Comment:
This is an incorrect use of CMake ABTYPE. Please set `SUBDIR=svxlink-$PKGVER/src` in `spec`.